pynuodb/cursor.py

```diff
@@ -130,6 +130,22 @@ def execute(self, operation, parameters=None):
         if self.rowcount < 0:
             self.rowcount = -1
         self.rownumber = 0
+        
+    def executesqltest(self, test):
```

Comment:
Is there a test for this function?  I don't see where executesqltest() is called.